accounting/fiscal_localizations/localizations/chile.rst

```diff
@@ -531,25 +507,24 @@ you might have and which is the related solution.
 - Error: Fecha/Número Resolucion Invalido RECHAZO- CAF Vencido : (Firma_DTE[AAAA-MM-DD] - 
   CAF[AAAA-MM-DD]) &gt; 6 meses.
   
-  *Hint:* Try to add a new CAF in the journal sequence related to this document as the one you’re 
-  using is expired. 
+  *Hint:* Try to add a new CAF related to this document as the one you’re using is expired.
 
 
 
 Credit Notes
 ------------
 
 When a cancellation or correction is needed over a validated invoice, a credit note must be
-generated. It is important to consider that a CAF file is required for the Credit Note sequence,
+generated. It is important to consider that a CAF file is required for the Credit Note,
 which is identified as document 64 in the SII.  
 
-.. image:: media/Chile25.png
```

Comment:
Is this file still used somewhere?

Reply:
No, it has been removed in the last commit